### ISSUE TITLE
bootstraps.gz file is not written to the aux_info/bootstrap directory in alignment mode

### ISSUE THREAD

**Issue by vasisht** [COMBINE-lab/salmon#123]
bootstraps.gz is being written to the directory from where salmon command is being run with salmon v0.8. It was being written to the correct directory with salmon v0.7. This happens with both bootstraps and Gibbs sampling. 

```
salmon quant --libType A  --targets transcriptome.fa --numGibbsSamples=50 --threads 6 --alignments transcriptome.bam --output salmon
```
With version 0.7

```
$ ls salmon/aux_info/bootstrap/
bootstraps.gz  names.tsv.gz
```

With version 0.8

```
$ ls
bootstraps.gz  salmon
$ ls salmon/aux_info/bootstrap/
names.tsv.gz
```

**Comment by rob-p** [comment]
Hi @vasisht,

  Can you tell me if you're seeing this same behavior in quasi-mapping mode?  I'm looking into this now.

--Rob

**Comment by vasisht** [comment]
Hi @rob-p,

quasi-mapping is writing the file to the correct directory. It's only an issue with the alignment mode. 

Thanks,
Vasisht

**Comment by rob-p** [comment]
Yup --- I see the source of the bug.  I will push a fix and cut a patch release.

**Comment by rob-p** [comment]
Hi @vasisht,

  Here is a pre-compiled binary of a hotfix that should resolve this issue.  The bug will be fixed in the 0.8.1 release that should appear by Monday.  However, there are one or two other things I want to include in that release, so I didn't tag another just yet.

--Rob

[Salmon-latest_linux_x86_64.tar.gz](https://github.com/COMBINE-lab/salmon/files/812143/Salmon-latest_linux_x86_64.tar.gz)

**Comment by vasisht** [comment]
@rob-p That works. Thanks for the quick fix!
